player/timeline/tl_matroska.c

```diff
@@ -281,7 +281,7 @@ static int find_ordered_chapter_sources(struct MPContext *mpctx,
             MP_INFO(mpctx, "Will scan other files in the "
                     "same directory to find referenced sources.\n");
             filenames = find_files(main_filename);
-            num_filenames = MP_TALLOC_ELEMS(filenames);
+            num_filenames = MP_TARRAY_AVAILABLE(filenames);
```

Comment:
This probably wants the number of found filenames, not how many can be stored (didn't look at `find_files` to see if it allocates exact amounts though).